.golangci.yml

```diff
@@ -47,7 +47,6 @@ linters:
     - goconst
     - gocritic
     - gocyclo
-    - godox
     - gofmt
     - goimports
     - gomnd
```

Comment:
why are you changing the golint config in this PR?

Reply:
2a1fd3c and 73e7c1d  reverted this